README.md

```diff
@@ -0,0 +1,112 @@
+
+```bash
+dcos package install dcos-enterprise-cli --cli --yes
+```
+
+### AWS config secret
+We already prepared the aws config for the application in our `main.tf`. Next we create the secret from it.
+
+```bash
+dcos security secrets create /instance-profile-app/aws-config -v "$(terraform output secret_aws_conf)"
+```
+
+### Install marathon-lb
+To access our app lets install marathon-lb. As we’re running strict mode we’ve to create a service-account and a service-account-secret
+
+#### Prepare service account and secret
+
+```bash
+dcos security org service-accounts keypair mlb-private-key.pem mlb-public-key.pem
+dcos security org service-accounts create -p mlb-public-key.pem -d "Marathon-LB service account" marathon-lb-sa
+dcos security secrets create-sa-secret --strict mlb-private-key.pem marathon-lb-sa marathon-lb/service-account-secret
+dcos security org users grant marathon-lb-sa dcos:service:marathon:marathon:services:/ read
+dcos security org users grant marathon-lb-sa dcos:service:marathon:marathon:admin:events read --description "Allows access to Marathon events"
+```
+
+#### Install marathon-lb
+```bash
+echo '{"marathon-lb": {"secret_name": "marathon-lb/service-account-secret","marathon-uri": "https://marathon.mesos:8443"}}' | dcos package install marathon-lb --options=/dev/stdin --yes
+```
+
+### Deploy the marathon app
+The last step is to finally deploy our simple app using the bucket we prepared. We’re using the template given in our terraform file. You can simply review it using terraform output marathon_app_definition.
+
+```bash
+terraform output marathon_app_definition | dcos marathon app add
+```
+
+Lets wait for the app to become healthy
+
+```bash
+until dcos marathon app show /instance-profile-app | jq -e .tasksHealthy==1 >/dev/null; do echo "waiting for app becoming healthy" && sleep 10;done
+```
+
+## Using the app
+Once the app is healthy we can post data with curl to it.
+
+```bash
+echo "foobar" | curl --user testuser:password -X POST -H "Host: binapp.mesosphere.com" -d @- $(terraform output public-agents-loadbalancer)/bin
+```
+
+This app is creating an ID for the posted content. With this ID its storing the content into the specified bucket. So we can now use the aws-cli to see if it worked for us. Please replace <id returned by the post> in the URL with the id you received from the command above.
+
+```bash
+aws s3 cp s3://$(terraform output s3_bucket_name)/bin/<id returned by the post> -
+```
+
+You see that there is a file in the s3 bucket and its content is the one we posted above.
+
+All this is based on Instance Profile and AssumeRole without any static credentials but the external_id which only works in combination the the Account and Role of our DC/OS cluster.
+
+You can tinker around with this technique and its even more valuable if you're running an AWS multi-account setup.
```

Comment:
All of this is based on Instance Profile and AssumeRole without any static credentials but the external_id which only works in combination the the Account and Role of our DC/OS cluster.

You can tinker around with this technique and it's even more valuable if you're running an AWS multi-account setup.